### PR TITLE
Add target="_blank" to links in MH and SG

### PR DESCRIPTION
**Pull request by PiotrKozlowski**
for: https://app.zenhub.com/workspaces/openstax-unified-5b71aabe3815ff014b102258/issues/openstax/unified/1229

### CODE REVIEW

File: src/app/content/studyGuides/components/__snapshots__/StudyGuides.spec.tsx.snap [hunk 1]
```diff
@@ -71,7 +71,7 @@ exports[`StudyGuides properly display summary highlights 1`] = `
             className="summary-highlight-content StudyGuidesListElement__HighlightContent-sc-5qgyda-2 jwHLiF"
             dangerouslySetInnerHTML={
               Object {
-                "__html": undefined,
+                "__html": "undefined",
```
Comment (janKozie1) [inline src/app/content/studyGuides/components/__snapshots__/StudyGuides.spec.tsx.snap:74]: Should this be changed? It renders a `"undefined"` string but normally the `highlightedContent` can't be `undefined`

Comment (PiotrKozlowski) [reply src/app/content/studyGuides/components/__snapshots__/StudyGuides.spec.tsx.snap:74]: I saw that and I think it doesn't really matter? I could change mock to contain something which will render this mocked value

Comment (janKozie1) [reply src/app/content/studyGuides/components/__snapshots__/StudyGuides.spec.tsx.snap:74]: I don't think it matters either, just looks bit strange. There is probably nothing wrong with adding data to mocks that should be there

**Review by janKozie1 — APPROVED** [review]

**Review by philschatz — APPROVED** [review]
Nice use of memoizing. I am not familiar with `React.useMemo` . Does it drop least-recently-used items?

**Comment by PiotrKozlowski** [comment]
Do you mean if the previous value is removed from the memory when deps changes?
I think yes, since they are stored in the array and overwritten on deps change https://github.com/facebook/react/blob/df14b5bcc163516fc0f1ad35e9b93732c66c1085/packages/react-dom/src/server/ReactPartialRendererHooks.js#L352

**Pull request merged**